test/test_vtk.py

```diff
@@ -0,0 +1,190 @@
+import pytest
```

Comment:
- [ ] Add type hints to improve readability and enable static type checking.
- [ ] Implement error handling with try-except blocks and logging.
- [ ] Refactor the verification loops into a separate function verify_vtk_files to reduce code duplication.
- [ ] Define constants for repeated string literals.
- [ ] Add logging statements to track the progress of the test.
- [ ] Include descriptive messages in assertions to make debugging easier.
- [ ] Use os.path.join() consistently for all file path constructions.